### ISSUE TITLE
Implementing Navigation component

### ISSUE THREAD

**Issue by lemarcque** [android10/Android-CleanArchitecture-Kotlin#81]
I heard about the Navigation Component part of the Android Jetpack.
Is it worth to use it ? and if yes, how could we apply it with this architecture because it slightly change the way we navigate between screens.

**Comment by Zhuinden** [comment]
It has some advantages and some disadvantages just like anything, really.

**Comment by sachin09999** [comment]
yes its worth of use and we can applly it by going through the documentation https://developer.android.com/develop/ui/compose/navigation

**Comment by Zhuinden** [comment]
Navigation3 will deprecate all of it anyway.
